Makefile

```diff
@@ -56,3 +56,14 @@ build-darwin-amd64:
 .PHONY: build-darwin-arm64
 build-darwin-arm64:
 	GOOS=darwin GOARCH=arm64 go build -ldflags "-X github.com/Azure/draft/cmd.VERSION=${DRAFT_VERSION}" -v -o ./bin/draft-darwin-arm64
+
+.PHONY: clean
```

Comment:
lets have this be a more descriptive name: `cleanup-entra-app`. That way if we ever have more cleanup commands we could group them under a single `clean` command

Reply:
yeah makes sense!